tests/tasks/create_test_interfaces_with_dhcp.yml

```
@@ -43,8 +43,15 @@
       nmcli d set testbr managed false
     fi
     ip link set testbr up
-    ip addr add 192.0.2.1/24 dev testbr
-    ip -6 addr add 2001:DB8::1/32 dev testbr
+    timer=0
+    while ! ip addr show testbr | grep -q 'inet [1-9]'
+    do
+        let "timer+=1"
+        if [ $timer -eq 30 ]; then break; fi
+        ip addr add 192.0.2.1/24 dev testbr
+        ip -6 addr add 2001:DB8::1/32 dev testbr
+        sleep 1
```

It seems my question why to add the addresses again and again was not clear. Maybe using code helps. Why is this not possible?

```suggestion
    ip addr add 192.0.2.1/24 dev testbr
    ip -6 addr add 2001:DB8::1/32 dev testbr
    timer=0
    while ! ip addr show testbr | grep -q 'inet [1-9]'
    do
        let "timer+=1"
        if [ $timer -eq 30 ]; then break; fi
        sleep 1
```